package.json

```
@@ -7,9 +7,13 @@
     "lint": "eslint --ext .ts,.vue src",
     "type-check": "tsc",
     "gen-api": "node ./scripts/gen-api.js",
-    "start-mock": "prism mock -p 8090 -d https://raw.githubusercontent.com/traPtitech/Emoine_R/main/docs/openapi.yml"
+    "start-mock": "prism mock -p 8090 -d https://raw.githubusercontent.com/traPtitech/Emoine_R/main/docs/openapi.yml",
+    "buf": "buf generate ../Emoine_R-server/api/proto/"
```

githubから取ってくるならこうするらしい
https://buf.build/docs/reference/inputs
```suggestion
    "buf": "buf generate https://github.com/traPtitech/Emoine_R.git#subdir=api/proto"
```

---

:tasukatta: